src/components/common/entities/WebViewWithInjected/index.tsx

```
@@ -25,14 +29,18 @@ type WebViewMessageEvent = NativeSyntheticEvent<WebViewMessage>;
 
 interface WebViewWithInjectedProps {
   source: WebViewSource;
-  onMessage?: <Data>(reqMessage: MessageEventRequestData<Data>) => void;
+  // onMessage?: <Data>(reqMessage: MessageEventRequestData<Data>) => void;
+  onMessage?: (
+    reqMessage: MessageEventRequestData,
+  ) => MessageEventResponseData | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)